src/components/Disputes/DisputeActions.js

```diff
@@ -286,6 +301,7 @@ const useInfoAttributes = (
     negativeBackground,
     phase,
     positiveBackground,
+    status,
     theme.accent,
```

Comment:
should this `theme.accent` be as a dependency here?

Reply:
Yes it should, the accent could change (e.g in a dark theme)

Reply:
nice